cmd/ebrelayer/types/types.go

```diff
@@ -185,6 +192,30 @@ type EthereumBridgeClaim struct {
 
 // ProphecyClaimUnique for data part of ProphecyClaim transaction in Ethereum
 type ProphecyClaimUnique struct {
-	CosmosSenderSequence *big.Int
-	CosmosSender         []byte
+	ProphecyID []byte
+}
+
+type CosmosSignProphecyClaim struct {
+	CosmosSender      sdk.ValAddress
+	NetworkDescriptor oracle.NetworkDescriptor
+	ProphecyID        []byte
+}
+
+// ProphecyInfo store all data needed for smart contract call
+type ProphecyInfo struct {
+	TokenAmount          big.Int
+	ProphecyID           []byte
+	EthereumAddresses    []string
```

Comment:
This should be renamed to `EthereumTokenAddress` or `TokenAddress`.

Reply:
it is signer's ethereum address, rename to EthereumSignerAddresses